modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc

```diff
@@ -1,42 +1,36 @@
 = File Firewall
 :toc: right
+:regex-info-url: http://www.regular-expressions.info/
+:supported-mimetype-list-url: https://github.com/owncloud/core/blob/master/resources/config/mimetypemapping.dist.json
 
 == Introduction
 
-The File Firewall GUI enables you to manage firewall rule sets. You can
-find it in your ownCloud admin page, under `Admin -> Security`. The File
-Firewall lets you control access and sharing in fine detail, by creating
-rules for allowing or denying access restrictions based on: _group_,
-_upload size_, _client devices_, _IP address_, _time of day_, as well as
-many more criteria. In addition to these restriction options, the File
-Firewall app also supports rules based on
-http://www.regular-expressions.info/[regular expressions].
+The File Firewall GUI enables you to manage firewall rule sets. 
+You can find it in your ownCloud admin page, under `Admin -> Security`. 
```

Comment:
`Admin -> Security` --> why not using the menu macro as we do in the other documents?

Reply:
Good point.

Reply:
Change made anyway.